posthog/hogql/modifiers.py

```diff
@@ -50,7 +51,11 @@ def set_default_modifier_values(modifiers: HogQLQueryModifiers, team: "Team"):
         modifiers.optimizeJoinedFilters = False
 
     if modifiers.bounceRatePageViewMode is None:
-        modifiers.bounceRatePageViewMode = BounceRatePageViewMode.COUNT_PAGEVIEWS
+        modifiers.bounceRatePageViewMode = (
+            BounceRatePageViewMode.UNIQ_URLS
+            if get_instance_region() == "EU"
+            else BounceRatePageViewMode.COUNT_PAGEVIEWS
```

Comment:
I'm just a bit worried if we make the numbers people see based on `UNIQ_URLS`, we'll never be able to go back to `COUNT_PAGEVIEWS` as the universal default, meaning the product working differently in one region forever. What's the plan for removing the special case ASAP?

Reply:
This is why we haven't made this change already. It'll go back to using COUNT_PAGEVIEWS as soon as v2 sessions is out